.jenkins/changeset-validation.Jenkinsfile

```diff
@@ -0,0 +1,16 @@
+pipeline {
+    libraries { lib 'adverity-shared-library@2.1.0' }
+    agent { label "PRJob" }
+    environment {
+        NODE_VERSION = getNodeVersion()
+        GITHUB_TOKEN = credentials('e8c4eee6-cef3-4fd5-a65c-1050f7ecb0c7')
+    }
+    stages {
+        stage('Validate changesets') {
+            steps {
+                validateChangesets()
```

Comment:
can we add the validation that every changeset needs an `issue` linked ?

Reply:
https://github.com/pmedianetwork/eslint-config/pull/56/commits/35031624d9b1565008e8bab2f32b410bd75a85f9